### ISSUE TITLE
iOS support

### ISSUE THREAD

**Issue by JonasVautherin** [mintware-de/flutter_p2p#1]
This seems like a pretty cool project!

I am not very familiar with WiFi Direct, but my understanding was that Apple does not support it (as in: it will not be possible to communicate between iOS and Android using this plugin). Is that correct?

**Comment by devtronic** [comment]
You're right, iOS only supports something called [MultipeerConnectivity](https://developer.apple.com/documentation/multipeerconnectivity). Maybe it's possible to use native WiFi functionality to connect both OS' (https://developer.apple.com/documentation/network)

**Comment by JonasVautherin** [comment]
Ok, thanks for the info!

**Comment by agnoam** [comment]
There's any progress about this subject ?

**Comment by JonasVautherin** [comment]
I believe that Apple just does not play with the others on the WiFi-Direct matters (surprised? :trollface:). They have their own thing. What's probably possible is to have one smartphone create an AP and let the others connect to it. But that's not WiFi-Direct.

**Comment by agnoam** [comment]
Hmmm.... a little bit complicated way.

**Comment by agnoam** [comment]
Is there no way to do something about it ?

**Comment by JonasVautherin** [comment]
Ask Apple to play with the others? :D

**Comment by agnoam** [comment]
Ha ha ha 😂
